packages/wallets/src/api/client.ts

```diff
@@ -5,44 +5,61 @@ import { SDK_NAME, SDK_VERSION } from "../utils/constants";
 
 import type {
     CreateWalletDto,
+    WalletV1Alpha2ErrorDto,
     WalletV1Alpha2ResponseDto,
     CreateTransactionDto,
     SubmitApprovalDto,
     WalletsV1ControllerGetTransaction4Response,
+    WalletsV1ControllerSubmitApprovals4Error,
+    WalletsV1ControllerGetTransaction4Error,
     WalletsV1Alpha2TransactionResponseDto,
+    WalletsV1ControllerCreateTransaction4Error,
+    WalletsV1ControllerGetSignature4Error,
     CreateSignatureRequestDto,
     WalletsV1Alpha2SignatureResponseDto,
+    WalletsV1ControllerCreateSignatureRequest4Error,
+    WalletsV1ControllerSubmitSignatureApprovals4Error,
     WalletsV1Alpha2TransactionsResponseDto,
+    WalletsV1ControllerGetTransactionsWithoutChain4Error,
     Nftevm,
     Nftsol,
+    FetchContentFromWalletError,
     WalletBalanceResponseDto,
+    BalanceControllerGetBalanceForLocator2Error,
     CreateSignerInputDto,
     DelegatedSignerDto,
+    WalletsV1ControllerCreateDelegatedSigner4Error,
+    WalletsV1ControllerGetDelegatedSigner4Error,
 } from "./gen/types.gen";
+import { InvalidApiKeyError } from "../utils/errors";
 
 type CreateWalletParams = CreateWalletDto;
-type CreateWalletResponse = WalletV1Alpha2ResponseDto;
-type GetWalletResponse = WalletV1Alpha2ResponseDto;
+type GetWalletSuccessResponse = WalletV1Alpha2ResponseDto;
```

Comment:
i think this is getting long enough where it would make more sense in a separate types file

Reply:
agree https://github.com/Crossmint/crossmint-sdk/pull/1009/commits/c78f549f4fc8dd77791b05509721e0e1c479ee07